Added mapper functionality to SlackV3 (#25666)
demisto/content: Packs/Slack/ReleaseNotes/3_1_30.md

#### Integrations

##### Slack v3
 

> - Fixed an issue when creating a new *incident* using the BOT fails to create an incident in XSOAR.

Reviewer: ```suggestion
- Fixed an issue when creating a new incident in XSOAR from the bot failed.
```
Is this the fix with the authorizations?

> - Added support for Mappers & Classifiers.

Reviewer: ```suggestion
- Added support for classification & mapping.
```
